### ISSUE TITLE
MAE values

### ISSUE THREAD

**Issue by Francesco-Carlucci** [inventec-ai-center/bp-benchmark#3]
Good afternoon,
I'm using your benchmark to compare some models with the data reported in your paper.

I've noted that the values of MAE reported in the benchmark are consistently larger than the one declared in the cited articles, even on the same dataset, for example in the case of U-Net.
I would like to ask you what may cause that? Do you compute the metric in a different manner?

Many Thanks!

**Comment by sergiogvz** [comment]
Hello Francesco, 

There are several reasons why this could happen and actually, this was one of our motivations for our benchmark. We have discussed several of them in our paper:

1. The models in the benchmark are trained only with PPG and without calibration or personalization, which would already affect the MAE results. For example, some of the cited papers also used ECG.
2. Even though we are using the same original datasets, different studies have different preprocessing procedures and criteria for discarding samples. In addition, practitioners often do not disclose preprocessed data, which hampers reproducibility. Therefore, we decided to publish our preprocessed data sets along with the benchmark.
3. One possible main reason is discussed in the paper's section "Data splitting: subject information and skewed blood pressure distributions". The conclusion of this section in our paper is the following: "When splitting the data into validation folds, the skewed BP distribution and subject information leakage are commonly overlooked in BP research [[11](https://www.nature.com/articles/s41597-023-02020-6#ref-CR11),[12](https://www.nature.com/articles/s41597-023-02020-6#ref-CR12),[20](https://www.nature.com/articles/s41597-023-02020-6#ref-CR20)], even discussing the latter as a calibration strategy [[20](https://www.nature.com/articles/s41597-023-02020-6#ref-CR20)]. We have empirically shown how overlooking them leads to misleading and over-optimistic results". The mentioned section shows how the MASE results are considerably lower when these aspects are ignored. The same pattern will be observed for MAE.

In the "Evaluation metrics" Section, we explain how we compute the metric and we offer the code. I hope you found my answer helpful.
